src/queries/consul/uploads/upload.js

```diff
@@ -0,0 +1,49 @@
+import * as FileSystem from 'expo-file-system';
+
+import { namespace, secrets } from '../../../config';
+import { getConsulAuthToken } from '../../../helpers';
+
+// https://docs.expo.io/versions/latest/sdk/filesystem/#filesystemuploadasyncurl-fileuri-options
+export const uploadAttachment = async (uri, resourceRelation, resourceType) => {
+  const token = await getConsulAuthToken();
+
+  const fetchObj = {
+    httpMethod: 'POST',
```

Comment:
shouldn't this be `method`? how come `httpMethod`? i think this happened as we called together. in other files we always have `method` so we should here and for volunteer, right?